pkg/cmd/resource/namespace.go

```diff
@@ -0,0 +1,88 @@
+package resource
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
```

Comment:
Sorry to bring up these minor things all the time, but unless you have a more cohesive convention for organizing these imports in mind, you might want to just stick to two groups: (1) built-ins (like `fmt`), and (2) everything else (like `github.com/...`).

(And there's a few other files below that could use the same treatment.)

Reply:
goimports is configured to use three groups:
1. built-ins
2. third-party packages
3. internal packages

i.e. all packages that start with `github.com/stripe/stripe-cli` are grouped separately at the end.

https://github.com/stripe/stripe-cli/blob/b489043a11b42ffae12a3a98dd01e122d89ce393/.golangci.yml#L6

Reply:
Ah, I see. Okay, just as long as we have a system.